### PR TITLE
Remove CAMERA_RADIUS_CURV

### PR DESCRIPTION
**Pull request by orelgueta**
The parameter itself is simply not necessary. To make a long story short, it's not the camera radius curvature but the mirror. It is used in testeff to calculate the effective mirror area, but instead of using that effective mirror area, we use the one derived from detailed ray tracing. Therefore, this parameter can simply be removed.

On the way there, a few other issues were "discovered" and fixed with the PR.

### CODE REVIEW

**Comment by orelgueta** [comment]
In the end all sorts of other things were included in this PR. It also fixes #241.

File: simtools/simtel/simtel_runner_ray_tracing.py [hunk 1]
```diff
@@ -237,7 +242,11 @@ def _checkRunResult(self, runNumber=None):
 
     def _isPhotonListFileOK(self):
         """Check if the photon list is valid,"""
+        nLines = 0
         with open(self._photonsFile, "r") as ff:
-            nLines = len(ff.readlines())
+            for _ in ff:
```
Comment (GernotMaier) [inline simtools/simtel/simtel_runner_ray_tracing.py:247]: Looks definitely more complicated - any particular reason why the old implementation didn't work?

**Review by GernotMaier — APPROVED** [review]
All good - please merge.

File: simtools/simtel/simtel_runner_ray_tracing.py (hunk 1 above)
Comment (orelgueta) [reply simtools/simtel/simtel_runner_ray_tracing.py:247]: It was significantly slower, it read the entire file (~2 million lines) just to check if it has more than a 100 lines.

**Pull request merged**